x-pack/test/accessibility/apps/maps.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const a11y = getService('a11y');
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+  const { common } = getPageObjects(['maps', 'common']);
+
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Maps', () => {
+    before(async () => {
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/maps.json'
+      );
+      await common.navigateToApp('maps');
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/fixtures/kbn_archiver/maps.json'
+      );
+    });
+
+    it('loads  maps workpads', async function () {
+      await retry.waitFor(
+        'maps workpads visible',
+        async () => await testSubjects.exists('itemsInMemTable')
+      );
+      await a11y.testAppSnapshot();
+    });
+
+    it('provides bulk selection', async function () {
+      await testSubjects.click('checkboxSelectAll');
+      await a11y.testAppSnapshot();
+    });
+
+    it('provides bulk delete', async function () {
+      await testSubjects.click('deleteSelectedItems');
+      await a11y.testAppSnapshot();
+    });
+
+    it('single delete modal', async function () {
+      await testSubjects.click('confirmModalConfirmButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('single cancel modal', async function () {
+      await testSubjects.click('confirmModalCancelButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('create new map', async function () {
+      await testSubjects.click('newItemButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('click map settings', async function () {
+      await testSubjects.click('openSettingsButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('map save button', async function () {
+      await testSubjects.click('mapSaveButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('map cancel button', async function () {
+      await testSubjects.click('saveCancelButton');
+      await a11y.testAppSnapshot();
+    });
+
+    it('map inspect button', async function () {
```

Comment:
@rashmivkulkarni can we load sample data, open  the maps then make the inspect panel run through a11y tests. Right now its empty. And this is what gets displayed when you have data - 
<img width="1792" alt="Screen Shot 2022-03-18 at 12 48 32 PM" src="https://user-images.githubusercontent.com/7074629/159046939-aea86fd4-4a63-4acc-acd2-501c05bf83a8.png">
<img width="1792" alt="Screen Shot 2022-03-18 at 12 48 38 PM" src="https://user-images.githubusercontent.com/7074629/159046972-039dee04-297c-41f3-9d25-bb0f5206b9fe.png">